applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS
+	bool "DVFS module"
+	depends on SOC_NRF54H20_CPUAPP
+	depends on CAF_BLE_COMMON_EVENTS || (BT_MAX_CONN = 1)
+	default y
+	help
+	  This option enable DVFS module which switches frequency and voltage according
+	  to application needs.
+
+if DESKTOP_DVFS
+
+config DESKTOP_DVFS_RETRY_BUSY_TIMEOUT_MS
+	int "Retry timeout"
+	default 2
```

Comment:
Comment after our last discussion during standup:
Do you want to eventually set `retry timeout` to `1` and `retry count` to `6` (to speed up applying DVFS frequency change on `-EBUSY` error)?